inc/datahandlers/user.php

```diff
@@ -868,6 +868,31 @@ function verify_language()
 	}
 
 	/**
+	 * Verifies if a style is valid for this user or not.
+	 *
+	 * @return boolean True when valid, false when invalid.
+	 */
+	function verify_style()
+	{
+		global $lang;
+
+		$user = &$this->data;
+
+		if($user['style'] && !defined('IN_ADMINCP'))
```

Comment:
Any reason why the ACP doesn't check permissions? As an admin I would like to get an error if I select an invalid theme for an user.